src_deno/request/payload/event.ts

```diff
@@ -1817,11 +1817,11 @@ export interface FileItem {
   size?: number;
   source_team?: string;
   subject?: string;
-  subtype?: string;
+  subtype?: _w?: number
```

Comment:
Can you revert the changes in this file? They look invalid.

Reply:
Yeah, was overzealous with the replace